### PR TITLE
Implementation of the new draft design

### PR DESCRIPTION
**Pull request by kiyaGu**

### CODE REVIEW

File: web/src/components/displayPost.js [hunk 1]
```diff
@@ -1,11 +1,30 @@
...
-import * as moment from "moment";
+import DisplayComment from "./displayComment";
 
 class DisplayPost extends Component {
   state = {
     updateTooltipOpen: false,
-    deleteTooltipOpen: false
+    deleteTooltipOpen: false,
+    loadComment: "",
+    commentButtonText: "fa fa-angle-double-down"
```
Comment (kkarimi) [inline web/src/components/displayPost.js]: a visual element is not really a natural `state`, it'd be better to make this code more readable and maintainable by using what this actually refers to in terms of application state like `showComments: true` and then in the `render() { ` before `return ( .. )` you can set the className like `if (this.state.showComment) { commentClass = '...' }`

File: web/src/components/displayPost.js [hunk 2]
```diff
@@ -24,155 +43,173 @@ class DisplayPost extends Component {
...
+              &nbsp;&nbsp;&nbsp;
+              {/* if the post has more than one tag/interest */}
+              <span>
+                {postData.interests.map(interest => interest.name).join(" / ")}
+              </span>
+            </p>
+          </Col>
+          <Col sm={2}>
+            <p className="feedCreatedAt">
+              {/* {moment(postData.createdAt).format("MMM Do, YYYY")}&nbsp; */}
```
Comment (kkarimi) [inline web/src/components/displayPost.js]: remove commented code

File: web/src/components/views/CreateUser.js [hunk 3]
```diff
@@ -9,48 +9,42 @@ import {
...
-                  name="userType"
-                  value="organisation"
-                  onChange={this.props.handleSelect}
+        {this.props.inputData.userType === "volunteer" ? (
+          <Fragment>
+            <Row>
+              <Col sm={4} />
+              <Col sm={8} xs={12} id="signUpFacebook">
+                <FacebookLogin
+                  appId="138462666798513"
```
Comment (kkarimi) [inline web/src/components/views/CreateUser.js]: better if this comes from `.env` e.g. `REACT_APP_FACEBOOK_APP_ID` which then you can read in the react app

File: web/src/containers/Feed.js [hunk 4]
```diff
@@ -118,6 +151,37 @@ class Feed extends Component {
...
+      fetch(`/api/post/${postID}/comment`, {
+        method: "POST",
+        body: JSON.stringify({ content: this.state.newComment }),
+        headers: {
+          Accept: "application/json",
+          "Content-Type": "application/json"
+        },
+        credentials: "same-origin"
+      }).then(() => {
+        this.getComments(postID);
```
Comment (kkarimi) [inline web/src/containers/Feed.js]: always tricky to use multiple async operations, but if this works fine then fine for now

File: web/src/containers/Feed.js [hunk 5]
```diff
@@ -31,13 +31,44 @@ class Feed extends Component {
...
+    newComment: "",
+    commentsDisplayed: false
   };
 
   componentWillMount() {
     this.getFeeds();
+    this.setState({
+      loadComment: false
+    });
+    this.forceUpdate();
```
Comment (kkarimi) [inline web/src/containers/Feed.js]: interesting :)

File: web/src/components/displayPost.js (hunk 1 above)
Comment (kiyaGu) [reply web/src/components/displayPost.js]: good point...will fix it

File: web/src/components/views/CreateUser.js (hunk 3 above)
Comment (kiyaGu) [reply web/src/components/views/CreateUser.js]: done

File: web/src/containers/Feed.js (hunk 4 above)
Comment (kiyaGu) [reply web/src/containers/Feed.js]: Yes, but in this case It is a process that runs under the hood and will not affect the displaying of the feed did it like this

**Pull request merged**